### PR TITLE
Adds docs around adding new components

### PR DESCRIPTION
**Pull request by ashfurrow**
After #1003 I wanted to make sure we document how to do this.

### CODE REVIEW

File: docs/adding_new_components.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+import { storiesOf } from "@storybook/react-native"
+import React from "react"
+import { MyNewComponent } from "../MyNewComponent"
+
+storiesOf("MyNewComponent").add("Show default component", () => {
+  return <MyNewComponent />
+})
+```
+
+Run `yarn storybook`. This will pick up the new storybook and modify the `storyLoader.js` to add it to Emission's storybooks. You should now be able to run the Emission app and see the component through your storybook browser.
```
Comment (orta) [inline docs/adding_new_components.md]: `yarn start` will run this for you, wouldn't recommend this ^

**Comment by orta** [comment]
That's the only feedback I got 👍

**Comment by ashfurrow** [comment]
Cool all done'd!

**Comment by orta** [comment]
Yep

**Pull request merged**